docs/INTRODUCTION.md

```diff
@@ -1,30 +1,24 @@
 
-##### History #####
+For more information on how containers or Docker works, we'll let [Docker do the talking](https://www.docker.com/resources/what-container).
+
+<img src="docker-containerized-appliction-blue-border_2.png" width="370"/>
+
+The Splunk user community has asked us to support containerization as a platform for running Splunk. The promise of running applications in a microservice-oriented architecture evangelizes the principles of infrastructure-as-code and declarative directives, and we aimed to bring those benefits with the work in this codebase. This project delivers on that request: to provide the rich functionality that Splunk Enterprise offers with the user-friendliness and production-readiness of container-native software.
+
+## History
 In 2015, Denis Gladkikh (@outcoldman) created an open-source GitHub repository for installing Splunk Enterprise, Splunk Universal Forwarder and Splunk Light inside containers.
-Universal Forwarders and standalone instances
-were being brought online at a rapid pace, which introduced a new level of complexity into the enterprise environment. 
-In 2018, a new container image was created to improve the flexibility with which Splunk Enterprise could be operated in larger and more dynamic environments.
-Splunk's new container can now start with a small environment and grow with the deployment. This however has caused a divergence
-from the open-source community edition of the Splunk Enterprise container. As a result, containers for Splunk Enterprise versions prior to 7.1 can not 
-be used with, or in conjunction with, this new version as it is not backward compatible. 
-We are also unable to support version updates from any prior container to the current version released with Splunk Enterprise and Splunk Universal Forwarder 7.2, as the older versions are not forward compatible. 
-We are sorry for any inconvenience this may cause. 
 
+Universal Forwarders and standalone instances were being brought online at a rapid pace, which introduced a new level of complexity into the enterprise environment. In 2018, a new container image was created to improve the flexibility with which Splunk Enterprise could be operated in larger and more dynamic environments. Splunk's new container can now start with a small environment and grow with the deployment. This however has caused a divergence from the open-source community edition of the Splunk Enterprise container. 
+
+As a result, containers for Splunk Enterprise versions prior to 7.1 can not  be used with, or in conjunction with, this new version as it is not backward compatible. We are also unable to support version updates from any prior container to the current version released with Splunk Enterprise and Splunk Universal Forwarder 7.2, as the older versions are not forward compatible. We are sorry for any inconvenience this may cause. 
```

Comment:
painfully obnoxiously minor, but unnecessary double spacing in `... not  be...`

Reply:
Done, thanks!